### ISSUE TITLE
Datetime error when create backfill

### ISSUE THREAD

**Issue by voducdan** [mage-ai/mage-ai#1907]
I create a backfill for my pipeline, after filling all fields, I got the error 
```
SQLite DateTime type only accepts Python datetime and date objects as input. [SQL: UPDATE backfill SET end_datetime=?, interval_type=?, interval_units=?, name=?, start_datetime=?, updated_at=CURRENT_TIMESTAMP WHERE backfill.id = ?] [parameters: [{'interval_units': '1', 'start_datetime': '2023-01-31 00:00:00', 'end_datetime': '2023-02-01 00:00:00', 'interval_type': 'day', 'name': 'backfill-load-dataset', 'backfill_id': 1}]]
```
It seems start_time and end_time is wrong format.
![image](https://user-images.githubusercontent.com/34914552/216104475-5fc12d93-fa72-4902-a106-7dc04c373b76.png)

 - OS: Ubuntu 20.04
 - Browser: chrome

**Comment by wangxiaoyou1993** [comment]
Hi @voducdan , which Mage version are you using? Could you try using latest docker image or upgrading your version to 0.7.86?

**Comment by wangxiaoyou1993** [comment]
@voducdan Have you got a chance to try latest docker image? Do you still have this error?

**Comment by wangxiaoyou1993** [comment]
I'm going to close this ticket. The issue has been fixed in version 0.7.85. Feel free to reopen the ticket if the bug happens again.
